6-AdvancedScenarios/1-call-api-obo/SPA/public/manifest.json

```diff
@@ -1,6 +1,13 @@
 {
   "short_name": "React App",
   "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.svg",
```

Comment:
`favicon.svg` seems missing in the files

Reply:
File dose [exists ](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/blob/basher-6-1/6-AdvancedScenarios/1-call-api-obo/SPA/public/favicon.svg)